### ISSUE TITLE
[Idea] EZ Surgery

### ISSUE THREAD

**Issue by Ebinmaymayoverlord** [Zaers/-tg-station#257]
Can we all agree that surgery was a mistake?It's mostly a huge amount of busywork that takes a goddamn while to be performed and hurts my hands.It's only redeemable part is how certain parts can be failed for cool effects(such as massive brain damage,or permanent genital multilation)
My idea is simple:Add a way to let us you do surgery with one button.It needs to have all the tools in on your person/tile and you can do all the surgery on someone else,just by doing something ala table crafting.
Of course,not to invalidate regular surgery,make this one take twice as long to be performed,and make it so that if you were to fail one step due to RNG because of improper tools,you fail the whole thing and waste your time.Also of course keep the regular surgery in.
Please,just add something like this and my now-developing carpal tunnel will be grateful.

**Comment by octareenroon91** [comment]
Yes that would be neat. I'll look at ways we can hook into the surgery system.

**Comment by Asd396** [comment]
Maybe we should look into porting https://github.com/tgstation/-tg-station/pull/12924 so we could hook it into tablecrafting?

**Comment by octareenroon91** [comment]
I have a feeling tablecrafting would give us some really screwed up bugs.

Maybe setup something where an operating computer and sleeper can be linked?

Carpal tunnel is the devil.

**Comment by Ebinmaymayoverlord** [comment]
Someone on thread suggested a machine that does the surgeries,could be easier and less buggy to implement than tablecrafting,as well as expanding the amount of original content our serber has.The sprite could be a modified sleeper.
We should make it so that it's never faster than regular surgery though,for balance reasons.

**Comment by octareenroon91** [comment]
Asd coded up a machine. I'm having trouble merging the code but I wanted to update you.
